.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--
+
+For bug reports, please follow the template below. For enhancement proposals, feel free
+to use whatever template makes sense.
+
+-->
+
+#### Minimal, reproducible code sample, a copy-pastable example if possible
+
+```python
+# Your code here
+
+```
+
+#### Problem description
+
+Explain why the current behavior is a problem, what the expected output/behaviour 
+is, and why the expected output/behaviour is a better solution.
+
+#### Version and installation information
+
+Please provide the following:
+
+* Value of ``zarr.__version__``
+* Value of ``numcodecs.__version__``
+* Version of Python interpreter
+* Operating system (Linux/Windows/Mac)
+* How Zarr was installed (e.g., "using pip into virtual environment", or "using conda")
+
+Also, if you think it might be relevant, please provide the output from ``pip list`` or
+``conda list`` depending on which was used to install Zarr.
```

Comment:
Might suggest `conda env export` and `pip freeze` instead. These produce files that can be used to immediately install a user's environment and begin playing with it. While `conda list` and `pip list` do provide the same info, there is no one liner that turns them into an environment on a dev's machine.

Reply:
Done.